gin-redis/structure/request/userFeedback.go

```diff
@@ -1,4 +1,5 @@
-package requestStruct
+// Package requeststruct contains structs for api request
+package requeststruct
 
```

Comment:
camel cases are a good way any way to do so

Reply:
linter shows error and ask not to name the package in camel case